script/install_tbb.sh

```diff
@@ -17,14 +17,22 @@ source ./script/set.sh
 # Install TBB
 if [ "$ALPAKA_CI_OS_NAME" = "Linux" ]
 then
-    travis_retry sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install libtbb-dev
+    travis_retry sudo apt-get -qqq update
+    travis_retry sudo apt-get install -y wget build-essential pkg-config cmake ca-certificates gnupg
+    travis_retry sudo wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+
+    sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+    travis_retry sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
+
+    travis_retry sudo apt-get update
+    travis_retry sudo apt-get install -y intel-oneapi-tbb-devel-2021.2.0
```

Comment:
The version number does not match the title of the PR.

Reply:
Thx! I had to change it after creating the PR, because cmake could not find 2021.1.1 yet. I think the full cmake support came in 2021.2.0.